spec/views/layouts/application.html.erb_spec.rb

```diff
@@ -10,8 +10,8 @@
     expect(rendered).to match t("layouts.application.faq_nav_link")
     # Footer is temporarily looking for contributors
     # expect(rendered).to match t("layouts.application.footer.text")
```

Comment:
Do we still need this assertion?

Reply:
Nice catch! I'm fine with removing these. @hantuzun thoughts?